www/main/navigator/shabad/ShabadContent.jsx

```diff
@@ -265,8 +265,19 @@ const ShabadContent = () => {
       const currentVerseIndex = mappedShabadArray.findIndex(
         ({ verseId }) => verseId === activeVerseId,
       );
-      let nextVerseIndex;
 
+      let nextVerseIndex = homeVerse;
+
+      if (intelligentSpacebar) {
+        nextVerseIndex = handleIntelligentSpacebar(nextVerseIndex, mappedShabadArray, currentVerseIndex);
+      }
+
+      const nextVerseId = mappedShabadArray[nextVerseIndex].verseId;
+      scrollToVerse(nextVerseId);
+      updateTraversedVerse(nextVerseId, nextVerseIndex);
+    }
+
+    function handleIntelligentSpacebar(nextVerseIndex, mappedShabadArray, currentVerseIndex) {
```

Comment:
Can you change this function definition to arrow-body function to keep it consistent with other functions.